content/04-class-field.md

```diff
@@ -393,6 +393,7 @@ The `extern` keyword causes the compiler to not generate the field in the output
 The `final` keyword can be used on class fields with the following effects:
```

Comment:
this `on class fields` doesn't sound accurate now (even before your change, but still)

Reply:
Ah good point, maybe I should move my line to outside of line 393, as in above it.